### PR TITLE
fix(vendors): concat the files properly

### LINKED ISSUE #162: Console errors on new projects
A couple of console errors seem to be appearing on new projects now:

image

@jkuri the second one seems to be related to the functionality you were working on, can you have a look?

### PR DESCRIPTION
**Pull request by cironunes**
Closes #162

### CODE REVIEW

**Comment by jkuri** [comment]
LGTM

File: lib/broccoli/angular2-app.js [hunk 1]
```diff
@@ -61,7 +61,17 @@ Angular2App.prototype.toTree = function() {
     });
 
     var thirdPartyJs = new Concat(thirdPartyJsTree, {
```
Comment (jkuri) [inline lib/broccoli/angular2-app.js]: Add this to vendorNpmJS instead of thirdPartyJs

**Comment by jkuri** [comment]
Please wait @filipesilva before merging, I gave this lgtm a little to soon.

File: lib/broccoli/angular2-app.js (hunk 1 above)
Comment (cironunes) [inline lib/broccoli/angular2-app.js]: my mistake, you're right.

**Comment by jkuri** [comment]
This now looks good.

**Comment by filipesilva** [comment]
Nice, http and router are also in!

**Pull request merged**